src/Data/Container/Combinator.agda

```diff
@@ -29,36 +29,68 @@ module _ {s p : Level} where
   id .Shape    = ⊤
   id .Position = F.const ⊤
 
+  to-id : ∀ {x} {X : Set x} → F.id X → ⟦ id ⟧ X
```

Comment:
Oh sorry, one more thing. By convention we tend to use `a b` and `A : Set a`, `B: Set b` for universe levels and types.